.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: ruby
 sudo: false
+cache: bundler
```

Comment:
Ok so you did `bundle update` instead of `rm Gemfile.lock`? That is a different way to how we're used to doing it. Will it definitely do the same thing? The [`bundle update` docs](https://bundler.io/man/bundle-update.1.html) talk about an `--all` option...

Reply:
OK, I didn't understand that bit about your message above. I'll do it the same way as our other repos.

Reply:
Think it's doing what we want now? Tentative ✅

Reply:
Yep, although I took the liberty of splitting into a separate commit.